shared/src/business/useCases/trialSessions/runTrialSessionPlanningReportInteractor.ts

```diff
@@ -97,14 +105,17 @@ export const getTrialSessionPlanningReportData = async ({
           .compareISODateStrings(a.startDate, b.startDate);
       });
 
-      const previousTermSessionList = [];
+      const previousTermSessionList: Object[] = [];
```

Comment:
Nit: looking at its usage on line 123 this might be a string[] instead of a Object[].

Reply:
Dang it, you're right! Wonder why it didn't give me a type error